app/graphql/mutations/vocabulary/create_vocabulary.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Mutations::Vocabulary::CreateVocabulary < Mutations::BaseMutation
+  argument :string_key, ID, required: true
+  argument :label, String, required: true
+  argument :locked, Boolean, required: false
+
+  field :vocabulary, Types::VocabularyType, null: true
```

Comment:
Same question here about when `vocabulary` would return null.  Since I've already posted a few comments about this, there may be something obvious that I'm not seeing, haha.  I won't post anymore comments about this in your other graphql classes.